spec/system/admin/bulk_order_management_spec.rb

```diff
@@ -1004,4 +1079,14 @@ def displays_default_orders
     expect(page).to have_selector "tr#li_#{li1.id}"
     expect(page).to have_selector "tr#li_#{li2.id}"
   end
+
+  def expect_line_items_results(line_items, excluded_line_items)
+    expect(page).to have_text "Loading orders"
```

Comment:
This can cause a flaky spec if the orders load quicker than this spec checks it.

Reply:
Yes, agree. I wanted to test the end of the search and check line items once it's ready. I don't know if I do prefer:
 - `expect(page).to have_text "Loading orders"`
 - `expect(page).not_to have_text "Loading orders"
 - or actually no check at all... 

What do you think?